hudi-client/pom.xml

```diff
@@ -85,6 +85,11 @@
       <groupId>log4j</groupId>
       <artifactId>log4j</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>${slf4j.version}</version>
```

Comment:
So where is the property `slf4j.version` definition?

Reply:
In the parent pom file，someone have already defined it before.

Reply:
Good, now I have no question.